modules/crud/sql/delete_test.go

```diff
@@ -0,0 +1,40 @@
+package sql
+
+import (
+	"context"
+	"testing"
+
+	"github.com/spaceuptech/space-cloud/model"
+)
+
+func TestGenerateDeleteQuery(t *testing.T) {
+	// truecases := 11
+	tests := []struct {
+		name, tableName, want string
+		req                   model.DeleteRequest
+	}{
+		{name: "Successfull Test", tableName: "fooTable", want: "DELETE FROM fooTable WHERE ((String1 = ?) AND (String2 = ?))", req: model.DeleteRequest{Find: map[string]interface{}{"String1": "1", "String2": "2"}}},
```

Comment:
output should be this : "DELETE FROM fooTable WHERE ((String1 = ?) AND (String2 = ?))"
but sometimes string1 & string2 interchange
"DELETE FROM fooTable WHERE ((String2 = ?) AND (String1 = ?))"

Reply:
The where clause parameters may get interchanged. Write your test cases to accommodate both the results using the or

Reply:
put an or clause. both the results are right